src/components/GameOver/GameOver.test.js

```diff
@@ -2,11 +2,20 @@ import React from 'react';
 import { shallow } from 'enzyme';
 import { GameOver } from './GameOver';
 
+const emptywinner = null
 const winner = { username: 'username' };
+const empty = []
+const drawerstest = [{username:'one'}, {username:'two'}] 
 
 describe('on render', () => {
```

Comment:
Can you make the describes more descriptive here, instead of `'on render'` maybe do `describe 'on winner'... it 'displays the winners username'` and `'describe 'on draw'... it 'displays the drawers'`?